src/main/java/uk/gov/justice/digital/hmpps/riskprofiler/services/NomisService.java

```diff
@@ -85,9 +85,8 @@ public void evictSocListAlertsCache(final String nomsId) {
         return getCandidates(uri);
     }
 
-    //@Cacheable("incident")
-    // TODO caching disabled for now as the initial query is too slow
-    public List<IncidentCase> getIncidents(@NotNull final String nomsId, @NotNull final List<String> incidentTypes, final List<String> participationRoles) {
+    @Cacheable(value = "incident", key = "#p0")
```

Comment:
is it safe to just cache using the first parameter?

Reply:
yes, the others are config